getting-started-guide/event-data.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Event data"
+description: "This page explains the fundamentals of timestamped event data in Axiom."
+sidebarTitle: Event data
+---
+
+Axioms’s mission is to help you leverage the power of timestamped event data.
+
+Axiom considers timestamped event data to be the record of every digital interaction between human, sensor, and machine. Event data is the atomic unit of activity. For this reason, every function in any business with digital activity can benefit from leveraging event data.
+
+Each event is simply a structured record—composed of key-value pairs—that captures meaningful interactions or changes in state within a system. While these can appear in various forms, they usually contain the following:
+
+- **Timestamp**: When the event occurred.
+- **Attributes**: A set of key-value pairs offering details about the event context.
+- **Metadata**: Contextual labels and IDs that connect related events.
+
+<Note>
+The timestamped event data that Axiom works with is not to be confused with time-series event data.
+</Note>
+
+## Uses of event data
+
+Event data, understood as the atomic unit of digital activity, is the lifeblood of modern businesses. Leveraging the power of event data is essential in the following areas, among others:
+
+- [Observability](/getting-started-guide/observability)
+- Security
+- Business intelligence
+- AI and machine learning
+- Finance
```

Comment:
Can we remove Finance and put Product analytics above business intelligence? That feels like a more natural selection and sequencing today. Sorry, not sure I can do multi-line suggestions inline in GitHub.